### DISCUSSION TITLE
Questions related to Real Estate project

### DISCUSSION THREAD

**Discussion by Jawwad-Fida** [Jawwad-Fida/Radon-Real-Estate-Business#4] [General]
Hi all!

If you have any questions or suggestion related to our Real Estate project, do let us know here.

Thank you!

**Comment by monivaa** [comment]
I think that if you need some help with such things then you definitely need to pay attention to one source that have already helped with everything if you need some [madrid new homes for sale](https://mlsworldwide.com/en/sale-spain-community-of-madrid-madrid/new-constructions) right now. I have already used a lot of different resources and I hope that this one will be a good one for you.
